### ISSUE TITLE
Review annotations to GO:0016401 palmitoyl-CoA oxidase activity  and GO:0016402 pristanoyl-CoA oxidase activity

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#5426]
Dear all, 

The proposal has been made to obsolete 

GO:0016401 palmitoyl-CoA oxidase activity (no xrefs)- 4 EXP (rat Acox1, rat Acaa1a, mouse Acadl, human ACAA1)
GO:0016402 pristanoyl-CoA oxidase activity (RHEA:40459) 

because they represent specific substrates.

**Comment by deustp01** [comment]
Hmm. The parent term, [GO:0003997](https://www.ebi.ac.uk/QuickGO/term/GO:0003997) acyl-CoA oxidase activity, which I guess is the intended replacement for GO:0016401 and 2, now has three children:
[GO:0044535](https://www.ebi.ac.uk/QuickGO/term/GO:0044535) very-long-chain fatty acyl-CoA oxidase activity
[GO:0016401](https://www.ebi.ac.uk/QuickGO/term/GO:0016401) palmitoyl-CoA oxidase activity
[GO:0016402](https://www.ebi.ac.uk/QuickGO/term/GO:0016402) pristanoyl-CoA oxidase activity

If the very-long-chain fatty acyl-CoA oxidase activity term is OK (seems OK to me), then as part of this clean-up, should it get some siblings for long-chain ~, medium-chain ~, short-chain ~, branched-chain ~, and so forth? @sjm41 ?

**Comment by sjm41** [comment]
>If the very-long-chain fatty acyl-CoA oxidase activity term is OK (seems OK to me), then as part of this clean-up, should it get some siblings for long-chain ~, medium-chain ~, short-chain ~, branched-chain ~, and so forth? @sjm41 ?

Right, that's basically what I proposed in https://github.com/geneontology/go-ontology/issues/28733:

```
GO:0003997 acyl-CoA oxidase activity (EC:1.3.3.6, RHEA:38959)
   |_GO:0044535 very-long-chain fatty acyl-CoA oxidase activity (RHEA:78847)
   |_NTR short-chain fatty acyl-CoA oxidase activity [RHEA:78859]
   |_NTR medium-chain fatty acyl-CoA oxidase activity [RHEA:78855]
   |_NTR long-chain-chain fatty acyl-CoA oxidase activity [RHEA:78851]
```
